src/GraphSDKGenerator/Options.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommandLine;
+
+namespace GraphSDKGenerator
+{
+    public enum VerbosityLevel
+    {
+        Minimal,
+        Info,
+        Debug, 
+        Trace
+    }
+
+    class Options
+    {
+        [Option('l', "language", Default = "CSharp", HelpText = "Desired output language")]
```

Comment:
`HelpText = "The target language for the generated code files. The values can be: Android, Java, ObjC, CSharp, PHP, Python, TypeScript, or GraphEndpointList"`